src/commands/dev/dev.js

```diff
@@ -463,6 +465,7 @@ const createDevCommand = (program) => {
     .option('-o ,--offline', 'disables any features that require network access')
     .option('-l, --live', 'start a public live session', false)
     .option('--functionsPort <port>', 'port of functions server', (value) => Number.parseInt(value))
+    .option('-e, --env <env>', 'set the env files priority', false)
```

Comment:
I'm not sure a flag is needed at the moment, so we can keep it out of scope for this PR.
The issue at hand is limited to having it configured in the `netlify.toml` `[dev]` block.

Reply:
@iib0011 what do you think about removing the `env` flag for now.
The issue is about configuring it in the `[dev]` block. Also if we add the flag we should add it in all relevant commands like `dev,dev:exec,functions:create,functions-serve`.

How about considering doing it in a separate PR?